unit_tests/test_aggregator.py

```diff
@@ -404,12 +366,9 @@ def test_delete_aggregator(aggregator_explicit):
 
 
 def test_add_to_batch_commands(aggregator_explicit):
-    # pylint: disable=protected-access
```

Comment:
Is it proper to move this to the top of he document even if it is only used on this test case?

Reply:
Actually I moved it to the top because we're now accessing several private members throughout the module _client_command_buffer, _all_uuids_in_selected_device_uuid_list, _post_request